rocket_twin/systems/ground.py

```diff
@@ -0,0 +1,25 @@
+from cosapp.base import System
+
+
+class Ground(System):
+    """Ground system that manages the stations.
+
+    Inputs
+    ------
+    stations : System,
+        space stations
+
+    Outputs
+    ------
+    """
+
+    def setup(self, stations=None):
+
+        if stations is None:
+            stations = []
+
+        self.add_property("stations", stations)
```

Comment:
could be removed. If you want to have access to the stations, you will just have to use : 

```
for station in self.children.values():
      do what you want with system child
```

Reply:
Check